doc/base/classes.xml

```diff
@@ -30129,10 +30129,10 @@
 </class>
 <class name="Node2D" inherits="CanvasItem" category="Core">
 	<brief_description>
-		Base node for 2D system.
+		A 2D game object, parent of all 2D related nodes. Has a position, rotation, scale and z-index.
```

Comment:
The rest of the docs seem to capitalize "Z-index", so maybe do it here too for consistency.